packages/synapse-react-client/src/components/SynapseNavDrawer.tsx

```diff
@@ -227,317 +230,319 @@ export const SynapseNavDrawer: React.FunctionComponent<
   }
 
   return (
-    <div className="SynapseNavDrawer">
-      <Drawer
-        variant="permanent"
-        className={`SynapseNavDrawerMenu ${isOpen ? 'tempDrawerOpen' : ''}`}
-      >
-        <div onClick={handleDrawerClose}>
-          <a
-            className="synapseIcon"
-            rel="noopener noreferrer"
-            href="/#!Home:0"
-            aria-label="Synapse Home"
-          >
-            <SynapseIconWhite />
-          </a>
-        </div>
-        <List>
-          {isLoggedIn && currentUserProfile && (
-            <>
-              <NavDrawerListItem
-                tooltip="Projects"
-                iconName="dashboard"
-                onClickOpenNavMenu={NavItem.PROJECTS}
-                isCurrentlySelectedItem={selectedItem == NavItem.PROJECTS}
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              <NavDrawerListItem
-                tooltip="Favorites"
-                iconName="favTwoTone"
-                onClickGoToUrl={`/#!Profile:${currentUserProfile.ownerId}/favorites`}
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              <NavDrawerListItem
-                tooltip="Teams"
-                iconName="peopleTwoTone"
-                onClickGoToUrl={`/#!Profile:${currentUserProfile.ownerId}/teams`}
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              <NavDrawerListItem
-                tooltip="Challenges"
-                iconName="challengesTwoTone"
-                onClickGoToUrl={`/#!Profile:${currentUserProfile.ownerId}/challenges`}
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              <NavDrawerListItem
-                tooltip="Download Cart"
-                iconName="download"
-                onClickGoToUrl="/#!DownloadCart:0"
-                badgeContent={numberOfFilesInDownloadList}
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              <NavDrawerListItem
-                tooltip="Trash Can"
-                iconName="delete"
-                onClickGoToUrl="/#!Trash:0"
-                handleDrawerClose={handleDrawerClose}
-                handleDrawerOpen={handleDrawerOpen}
-              />
-              {currentUserBundle?.isARReviewer && (
+    <>
```

Comment:
The success alert is created within the `CreateProjectModal`, which was included within the `.SynapseNavDrawer` div. There is a [background color](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/e1bfaaf8a450406096dbf48ac26e6407092b1ad4/packages/synapse-react-client/src/style/components/_synapse-nav-drawer.scss#L31C3-L31C3) applied to `.MuiPaper-root` within `.SynapseNavDrawer`. When the success alert was refactored to a MUI Alert, the background color applied to the alert as well, since the MUI Alert is based on the MUI Paper component.

The only change here is adding a wrapping empty JSX tag, so the `CreateProjectModal` can be moved out  of the SynapseNavDrawer div and the background color will no longer apply to the alert.